assets/blocks/learner-courses-block/learner-courses-edit.js

```diff
@@ -200,6 +201,27 @@ const LearnerCoursesEdit = ( {
 					progressBarBorderRadius: `${ options.progressBarBorderRadius }px`,
 				} }
 			>
+				<div className="wp-block-sensei-lms-learner-courses__deprecation-notice">
+					<Notice
+						status="warning"
+						isDismissible={ false }
+						actions={ [
+							{
+								label: __( 'Read more', 'sensei-lms' ),
+								onClick: () => {
```

Comment:
It has a `url` option, but it doens't support opening in a new window, so I used the `onClick` with `window.open`.